trieste/acquisition/optimizer.py

```diff
@@ -490,10 +490,10 @@ def _objective_value_and_gradient(x: TensorType) -> Tuple[TensorType, TensorType
     return (successes, fun_values, chosen_x, nfev)
 
 
-class ScipyLbfgsBGreenlet(gr.greenlet):  # type: ignore[misc]
+class ScipyOptimizerGreenlet(gr.greenlet):  # type: ignore[misc]
     """
-    Worker greenlet that runs a single Scipy L-BFGS-B. Each greenlet performs all the L-BFGS-B
-    update steps required for an individual optimization. However, the evaluation
+    Worker greenlet that runs a single Scipy L-BFGS-B (by default). Each greenlet performs all the
```

Comment:
There should really be some sort of unit test for `optimizer_args`, though it looks like currently there isn't. Do you think you could add something simple to test_optimizer (maybe passing through the args to `generate_continuous_optimizer` and checking that they get used or defaulted as appropriate)? Alternatively if you think this might be tested by a future PR then it might be ok to skip for now.

Reply:
This was going to be tested in a future PR. However, I'll add something now. It makes sense.